doc/api/child_process.md

```diff
@@ -1368,6 +1375,34 @@ It is also recommended that any `'message'` handlers in the child process
 verify that `socket` exists, as the connection may have been closed during the
 time it takes to send the connection to the child.
 
+### `subprocess.signalCode`
+
+* {integer}
+
+The `subprocess.signalCode` property indicates the signal number received by
+the child process if any, else null.
+
+### `subprocess.spawnfile`
+
+* {string}
+
+The `subprocess.spawnfile` property indicates the executable file name of
+the child process that is launched.
+
+For [`child_process.fork()`][], its value will be equal to
+[`process.execPath`][].
+For [`child_process.spawn()`][], its value will be the name of
+the executable file.
+For [`child_process.exec()`][],  its value will be the name of the shell
+in which the child process is launched.
+
+### `subprocess.spawnargs`
```

Comment:
`spawnargs` should go before `spawnfile` rather than after it.

Reply:
@Trott, thanks. Placed it in right place.